src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowOperationVisitor.cs

```diff
@@ -223,8 +223,6 @@ protected DataFlowOperationVisitor(TAnalysisContext analysisContext)
                 getInterproceduralCallStackForOwningSymbol: GetInterproceduralCallStackForOwningSymbol);
         }
 
-        public override int GetHashCode() => HashUtilities.Combine(GetType().GetHashCode(), DataFlowAnalysisContext.GetHashCode());
```

Comment:
I believe this is overridden in other visitors (sub-types) too. Can you check?

Reply:
Oops, yes, thanks.  Will also remove from others.

Any reason to think this change will introduce issues with DisposeAnalysis or ParameterValidationAnalysis?

Reply:
Yes, that is the primary thing I need to think on, as the operation visitors for those two are stateful. Let me get back to you early tomorrow morning.